example/mapboxExampleCamera.js

```diff
@@ -0,0 +1,284 @@
+import * as THREE from 'three';
+
+const utils = {
+	clamp: function ( n, min, max ) {
+
+		return Math.min( max, Math.max( min, n ) );
+
+	},
```

Comment:
This function is redundant to three.js' `MathUtils.clamp` function

Reply:
ok